public/themes/custom/hdbt_subtheme/src/scss/06_components/paragraphs/_job-listing-search.scss

```diff
@@ -0,0 +1,35 @@
+.component--job-listing-search {
+  background-color: $color-silver-light;
+}
+
+// If the district and project search is just before footer it needs to function the same way as unit search.
```

Comment:
Remove the District and project related comments also from here :)

Reply:
Nice catch! Fixed :)